packages/appstore/src/components/wallets-consent-popup/wallets-consent-form.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```

Comment:
I suggest to rename the file to it's modal's name : `ready-to-upgrade-wallets`

Reply:
Done 😃 👍🏼